Telerik.Examples.Mvc/NuGet.config

```diff
@@ -1,9 +1,10 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
     <packageSources>
         <clear /> <!-- ensure only the sources defined below are used -->
 		<add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
         <add key="Telerik.UI.for.AspNet.Core.Trial.2022.2.510" value=".\\nuget" />
```

Comment:
You should include the `Telerik.Web.Spreadsheet.Trial.2023.1.426` as a key-value pair as well. The IDE will predominantly look for the file in the specified `value` location.